modules/ncml_module

```diff
@@ -1 +1 @@
-Subproject commit 92e95489cdf742ef807a753a09f858a6b44c87e5
```

Comment:
Same question as above, curious why ncml_module is a dependency, submodule in this case, for the BES? Same for xml_data_handler